common/src/main/java/com/wynntils/core/features/Feature.java

```diff
@@ -48,10 +39,15 @@ public Feature() {
         }
     }
 
-    protected void init(
-            ImmutableList.Builder<WebProviderSupplier> apis,
-            ImmutableList.Builder<KeySupplier> keybinds,
-            ImmutableList.Builder<Condition> conditions) {
+    protected void init(ImmutableList.Builder<Condition> conditions) {
```

Comment:
You can make a method abstract to force subclasses to implement it...

Reply:
I want to make it optional to extend